server/datastore/mysql/apple_mdm.go

```diff
@@ -1642,6 +1642,36 @@ func (ds *Datastore) GetMDMAppleBootstrapPackageBytes(ctx context.Context, token
 	return &bp, nil
 }
 
+func (ds *Datastore) GetMDMAppleBootstrapPackageSummary(ctx context.Context, teamID uint) (fleet.MDMAppleHostStatusSummary, error) {
+	stmt := `
+          SELECT
+              COUNT(IF(ncr.status = 'Acknowledged', 1, NULL)) AS applied,
+              COUNT(IF(ncr.status = 'Error', 1, NULL)) AS failed,
+              COUNT(IF(ncr.status IS NULL, 1, NULL)) AS pending
+          FROM
+              hosts h
+          LEFT JOIN host_mdm_apple_bootstrap_packages hmabp ON
+              hmabp.host_uuid = h.uuid
```

Comment:
Could it happen that the same host gets multiple bootstrap packages? The name sounds like no (you only bootstrap once, right? :D ), but is that guaranteed? The hmabp has a primary key on host_uuid + command_uuid, so it would happily insert more than one row per host, and I'm not sure but I think `TokenUpdate` (where `RecordHostBootstrapPackage` gets called) can get called multiple times?